src/ansys/simai/core/data/projects.py

```diff
@@ -205,6 +205,14 @@ def compute_gc_formula(
 
         return gc_compute.result if gc_compute.is_ready else None
 
+    def cancel_training(self):
+        """Cancel training if there is one in progress."""
+
+        project_response = self._client._api.get_project(self.id)
```

Comment:
```suggestion
        self.reload()
```

But I think we should keep it simple and if we want the API to return if there actually was a training that got cancelled then we should do that instead of hacking around in the SDK.